src/main/resources/com/google/api/codegen/nodejs/method_sample.snip

```diff
@@ -45,7 +45,7 @@
 
 @private sampleWriteStreamingRequest(apiMethod)
     // Write request objects.
-    stream.write({@sampleMethodCallArgList(apiMethod)});
+    stream.write(request);
```

Comment:
Is there any way to obtain the name of this from `apiMethod` instead of having `request` always?